docs/migrate/migrate-setup.rst

```diff
@@ -0,0 +1,182 @@
+Migrate setup from ``qiskit-ibmq-provider``
+==============================================
+
+This guide describes how to migrate code from the legacy IBMQ provider (`qiskit-ibmq-provider`) package to use Qiskit Runtime (`qiskit-ibm-runtime`). This guide includes instructions to migrate legacy runtime programs to the new syntax. However, the ability to use custom uploaded programs is pending deprecation, so these should be migrated to use primitives instead.  
+
+- For instructions to use Qiskit Runtime primitives, see the `migration guide <migrate-guide.html>`__.  
+- For further details about migrating from `qiskit-ibmq-provider` to `qiskit-ibm-runtime`, see `Migration guide from qiskit-ibmq-provider <https://qiskit.org/documentation/partners/qiskit_ibm_runtime/migrate_from_ibmq.html>`__.
+
+ 
+Import path
+-------------
+
+The import path has changed as follows:
+
+**Legacy**
+
+.. code-block:: python
+
+    from qiskit import IBMQ
+
+**Updated**
+
+.. code-block:: python
+
+    from qiskit_ibm_runtime import QiskitRuntimeService
+
+Save and load accounts
+------------------------------------
+
+Use the updated code to work with accounts.
+
+**Legacy - Save accounts**
+
+.. code-block:: python
+
+    IBMQ.save_account("<IQX_TOKEN>", overwrite=True)
+
+**Updated - Save accounts**
+The new syntax accepts credentials for two different channels. For more information on retrieving account credentials, see the `getting started guide <https://qiskit.org/documentation/partners/qiskit_ibm_runtime/getting_started.html>`_.
+.. code-block:: python
+
+    # IBM cloud channel
+    QiskitRuntimeService.save_account(channel="ibm_cloud", token="<IBM Cloud API key>", instance="<IBM Cloud CRN>", overwrite=True)
+
+    # IBM quantum channel
+    QiskitRuntimeService.save_account(channel="ibm_quantum", token="<IQX_TOKEN>", overwrite=True)
+
+**Legacy - Load accounts**
+
+.. code-block:: python
+
+    IBMQ.load_account()
+
+**Updated - Load accounts**
+
+The new syntax combines the functionality from ``load_account()`` and ``get_provider()`` in one statement. The ``channel`` input parameter is optional. If multiple accounts have been saved in one device and no ``channel`` is provided, the default is ``"ibm_cloud"``.
+
+.. code-block:: python
+
+    # To access saved credentials for the IBM cloud channel
+    service = QiskitRuntimeService(channel="ibm_cloud")
+
+    # To access saved credentials for the IBM quantum channel
+    service = QiskitRuntimeService(channel="ibm_quantum")
+
+
```

Comment:
Another thing I really like about the new service, is that you can name your saved credentials. For example, I can do save 2 different h/g/p using different names:

```
QiskitRuntimeService.save_account(channel="ibm_quantum", token="<IQX_TOKEN>", instance="h1/g1/p1", name="premium")
QiskitRuntimeService.save_account(channel="ibm_quantum", token="<IQX_TOKEN>", instance="h2/g2/p2", name="open")
```

Then just load by name:

```
service = QiskitRuntimeService(name="open")
```